src/dotnet/MonoGame.Extended.Collisions/MonoGame.Extended.Collisions.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
It's probably worth trying to undo these changes and then have it squash-merged to minimise chaanges to the code, especially as it seems only a whitespace change